src/prommis/uky/uky_flowsheet.py

```diff
@@ -862,5 +874,782 @@ def display_results(m):
+        m.fs.reagents,
+        m.fs.solid_waste,
+        m.fs.precipitate,
+        m.fs.dust_and_volatiles,
+        m.fs.power,
+    ]
+
+    # define product flowrates
+    # TODO: Replace with flowsheet values, but what's the difference between pure and mixed? Product vs dust?
+
+    pure_product_output_rates = {
+        "Sc2O3": 1.9 * units.kg / units.hr,
+        "Dy2O3": 0.4 * units.kg / units.hr,
+        "Gd2O3": 0.5 * units.kg / units.hr,
+    }
+
+    mixed_product_output_rates = {
+        "Sc2O3": 0.00143 * units.kg / units.hr,
+        "Y2O3": 0.05418 * units.kg / units.hr,
+        "La2O3": 0.13770 * units.kg / units.hr,
+        "CeO2": 0.37383 * units.kg / units.hr,
+        "Pr6O11": 0.03941 * units.kg / units.hr,
+        "Nd2O3": 0.17289 * units.kg / units.hr,
+        "Sm2O3": 0.02358 * units.kg / units.hr,
+        "Eu2O3": 0.00199 * units.kg / units.hr,
+        "Gd2O3": 0.00000 * units.kg / units.hr,
+        "Tb4O7": 0.00801 * units.kg / units.hr,
+        "Dy2O3": 0.00000 * units.kg / units.hr,
+        "Ho2O3": 0.00000 * units.kg / units.hr,
+        "Er2O3": 0.00000 * units.kg / units.hr,
+        "Tm2O3": 0.00130 * units.kg / units.hr,
+        "Yb2O3": 0.00373 * units.kg / units.hr,
+        "Lu2O3": 0.00105 * units.kg / units.hr,
+    }
+
+    m.fs.costing.build_process_costs(
+        # arguments related to installation costs
+        piping_materials_and_labor_percentage=20,
+        electrical_materials_and_labor_percentage=20,
+        instrumentation_percentage=8,
+        plants_services_percentage=10,
+        process_buildings_percentage=40,
+        auxiliary_buildings_percentage=15,
+        site_improvements_percentage=10,
+        equipment_installation_percentage=17,
+        field_expenses_percentage=12,
+        project_management_and_construction_percentage=30,
+        process_contingency_percentage=15,
+        # argument related to Fixed OM costs
+        nameplate_capacity=500,  # short (US) ton/hr
+        labor_types=[
+            "skilled",
+            "unskilled",
+            "supervisor",
+            "maintenance",
+            "technician",
+            "engineer",
+        ],
+        labor_rate=[24.98, 19.08, 30.39, 22.73, 21.97, 45.85],  # USD/hr
+        labor_burden=25,  # % fringe benefits
+        operators_per_shift=[4, 9, 2, 2, 2, 3],
+        hours_per_shift=hours_per_shift,
+        shifts_per_day=shifts_per_day,
+        operating_days_per_year=operating_days_per_year,
+        pure_product_output_rates=pure_product_output_rates,
+        mixed_product_output_rates=mixed_product_output_rates,
+        mixed_product_sale_price_realization_factor=0.65,  # 65% price realization for mixed products
+        # arguments related to total owners costs
+        land_cost=m.fs.land_cost,
+        resources=resources,
+        rates=rates,
+        # TODO: Verify this cost
+        prices={
+            "dummy": 1 * getattr(units, "USD_" + CE_index_year) / units.kg,
+        },
```

Comment:
Should probably rename this from "dummy", but do we know what this cost actually should be?

Reply:
You could leave this argument as None or remove it if the chemicals list is entirely components with built-in prices (see https://github.com/prommis/prommis/blob/1d05932488e348a572f315969c98abb9b70dd164/src/prommis/uky/costing/ree_plant_capcost.py#L1997). I'm currently working on support for passing the total chemical cost or total waste cost as expressions similar to the land cost, and plan to have a PR up for that shortly.